lib/secure-boot-lib.robot

```diff
@@ -148,7 +148,7 @@ Check Secure Boot In Linux
     # secureboot: Secure boot disabled
     # or just:
     # Secure boot disabled
-    ${out}=    Execute Command In Terminal    dmesg | grep "Secure boot"
+    ${out}=    Execute Command In Terminal    dmesg | grep "Secure boot" | grep -v "Bluetooth"
```

Comment:
Nice catch, but please add a comment why this is needed, otherwise someone may try to remove it in the future.

Reply:
@JanPrusinowski Likely we need this to be ported to: https://github.com/Dasharo/open-source-firmware-validation/commits/regression-msi/

Reply:
@JanPrusinowski please link where changes were added.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/commit/133b843580206f791fd72831bbb26e02ecfbaf75